ecosystem/platform/server/app/models/user.rb

```diff
@@ -100,9 +100,7 @@ def add_oauth_authorization(data)
                         })
     when 'discord'
       raw_info = data['extra']['raw_info']
-      auth = auth.merge({
-                          username: "#{raw_info['username'].downcase}##{raw_info['discriminator']}"
-                        })
+      auth = auth.merge({ username: raw_info['username'].downcase })
```

Comment:
we still want the username on the authorization to remain the full thing with discriminator, so we can see the full thing in admin UI

Reply:
Oh right, this is authorization.username, not user.username. Reverted.